### PR TITLE
Update: provide alternative heading for AT (fixes #173)

### LINKED ISSUE #173: Missing notify heading label for dialogs
## Bug Description:
Depending on the design, certain dialogues throughout an Adapt experience often do not have a meaningful label associated with them. For instance:
* Transcripts
* Question Feedback
* Hotgraphic pop-ups

## Steps to Reproduce:
Take away any particular notify title
On entry into a modal dialog, note the dialog title.

## Actual Behavior:
Dialog with aria-labelledby referencing an invalid id:
`<div role="dialog" aria-labelledby="notify-heading" aria-modal="true" class="notify is-visua11ysettings" style="visibility: visible;">`

## Expected Behavior:
My proposal is to at least include an invisible notify-heading able to be read by a screen reader that describes the purpose of the pop-up. And any designed title would override that default if added.

### PR DESCRIPTION
**Pull request by chris-steele**
[//]: # (Link the PR to the original issue)
fixes https://github.com/adaptlearning/adapt-contrib-core/issues/173

### Update
* Provide an alternative heading for Assistive Technology if no visual heading is required

### CODE REVIEW

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

File: templates/notifyPopup.hbs [hunk 1]
```diff
@@ -19,6 +19,14 @@
...
           </div>
         </div>
+        {{else}}
+          {{#if altTitle}}
+          <div class="notify__title aria-label" id="notify-heading">
+            <div class="notify__title-inner" {{{a11y_attrs_heading 'notify'}}}>
+              {{{compile altTitle}}}
+            </div>
+          </div>
+          {{/if}}
```
Comment (oliverfoster) [inline templates/notifyPopup.hbs:29]:
```suggestion
        {{else if altTitle}}
        <div class="notify__title aria-label" id="notify-heading">
          <div class="notify__title-inner" {{{a11y_attrs_heading 'notify'}}}>
            {{{compile altTitle}}}
          </div>
        </div>
```
Reference https://handlebarsjs.com/guide/block-helpers.html#conditionals

**Review by oliverfoster — CHANGES_REQUESTED** [review]

File: templates/notifyPopup.hbs [hunk 2]
```diff
@@ -19,6 +19,12 @@
             {{{compile title}}}
           </div>
         </div>
+        {{else if altTitle}}
+          <div class="notify__title aria-label" id="notify-heading">
+            <div class="notify__title-inner" {{{a11y_attrs_heading 'notify'}}}>
```
Comment (oliverfoster) [inline templates/notifyPopup.hbs:24]: a11y_ helper thing again

**Review by oliverfoster — APPROVED** [review]

**Review by joe-replin — APPROVED** [review]
Tested in v5.19.6 of the FW and working well.

**Pull request merged**